src/setupTest.js

```diff
@@ -2,8 +2,8 @@ import React from "react";
 import { shallow, render, mount } from "enzyme";
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import fetch from './unit-test/mocks/fetch'
-// import sinon from "sinon";
+import fetch from './unit-test/mocks/fetch';
```

Comment:
It is unclear from the name that `fetch` is a mock.  Can we change the name to reflect what it does?

Reply:
No, it cannot be called differently because it refers to the real fetch method that the component is calling